packages/components/src/color-palette/index.tsx

```diff
@@ -249,7 +249,10 @@ function UnforwardedColorPalette(
 	};
 
 	const actions = !! clearable && (
-		<CircularOptionPicker.ButtonAction onClick={ clearColor }>
+		<CircularOptionPicker.ButtonAction
+			onClick={ clearColor }
+			disabled={ ! value }
```

Comment:
Let's also add `accessibleWhenDisabled` prop here.

Reply:
Thanks for the review. I've added it. I also applied the same logic to the `GradientPicker` and `DuotonePicker`.

Reply:
> Let's also add `accessibleWhenDisabled` prop here.

Thanks for catching that.